ClientCollection/Organization.ts

```diff
@@ -44,38 +44,18 @@ export class Organization extends rest.Collection<gracely.Error> {
 			)
 		return result
 	}
-	async changeName(
+	async update(
 		id: userwidgets.Organization.Identifier,
-		organization: userwidgets.Organization.Creatable,
+		organization: userwidgets.Organization.Changeable,
 		application: userwidgets.Application.Identifier
 	): Promise<userwidgets.Organization | gracely.Error> {
 		const entityTag = this.entityTags.organization[id]
-		const result = await this.client.put<userwidgets.Organization>(
-			`${this.configuration.pathPrefix}/organization/${id}/name`,
+		const result = await this.client.patch<userwidgets.Organization>(
```

Comment:
This is not the correct response type. check the API for the correct response. This also means that the return type of the function is wrong.